Hestia/app/src/main/res/values/strings.xml

```diff
@@ -49,4 +49,7 @@
     <string name="passSet">Password set to: </string>
     <string name="userSet">Username set to: </string>
     <string name="userNotSet">Username not changed (length under 5)</string>
+    <string name="cancelNameChange">Name of the device was not changed</string>
+    <string name="titleNameChange">Change name</string>
+    <string name="cancelIPChange">\"Cancel pressed\"</string>
```

Comment:
"Cancel pressed" is quite generic, so I believe we should not give it a specific name as "cancelIPChange". I think that "cancelPressed" would do. Or maybe change the string from "Cancel Pressed" to something more related to the IP.